src/main/java/homework_2/RandomCharsTable.java

```diff
@@ -0,0 +1,85 @@
+package homework_2;
+
+import java.io.BufferedReader;
+import java.io.IOException;
+import java.io.InputStreamReader;
+import java.util.Random;
+
+public class RandomCharsTable {
```

Comment:
3/5. It works, but it takes a significant amount of time to understand how. Please simplify.